ebb_lint/test/test_flake8.py

```diff
@@ -76,276 +76,277 @@ def scan_for_checkers(request):
 
 
 all_sources = [
-    '',
+    pytest.param(s) for s in [
+        '',
 
-    """
+        """
```

Comment:
Lots of retabbing because the change is from:

    all_sources = [
        '....'
    ]

to

    all_sources = [
        pytest.param(s) for s in [
            '...',
        ]
    ]